modules/simple_forms_api/app/models/simple_forms_api/vba_21_4140.rb

```diff
@@ -30,6 +30,14 @@ def dob
       [trimmed_dob&.[](0..3), trimmed_dob&.[](4..5), trimmed_dob&.[](6..7)]
     end
 
+    def employed?
+      data['employers'] ? data['employers'].size.positive? : false
+    end
+
+    def employment_history
+      [*0..3].map { |i| FormEngine::EmploymentHistory.new(data['employers'][i]) }
+    end
```

Comment:
It appears that you're handling the case where data['employers'] has fewer than four "real" elements. I see the test data explicitly is set up this way, with the fourth element being `{}`. Very nicely done there. 

But is this reflective of how data will actually come over from the front end? I'm not sure it is. I jumped over to the vets-website app to test this out, and it _appears_ to me that there will only be as many items as are actually added (won't be padded with blank entries up to the limit).

![image](https://github.com/user-attachments/assets/4d7b2abd-8f6c-4a04-a1c4-994fce5e72e0)

There are a lot of moving pieces here, and I may have missed something, so this is more of a "Hey, this caught my eye. Can you tell me if my curiosity is wrong?" type of inquiry.

Reply:
I can confirm through local testing that when employment history is less than four entries we get the data we want regardless of whether the missing employment history is an empty object or completely missing from the array. I can adjust the fixture to better match the data structure we get from prod, if that would be helpful. I believe I made the fixture the way I did to highlight that there is not a fourth employer, but it may have created more confusion than clarity.

Reply:
Awesome. Thanks. Approving.